### PR TITLE
Hookup front end

### PR DESCRIPTION
**Pull request by elizabethcan**
This branch hooks up the front end to the new back end and removes the unused routes. I had to do a little bit of tweaking on the front end and would like to add more functionality to the front end if time permits.

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-// import React from 'react';
+import ReactDOM from 'react-dom';
+import React from 'react';
 
-// ReactDOM.render(<App houseId="100" />, document.getElementById('app'))
+ReactDOM.render(<App houseId="100" />, document.getElementById('app'))
 
-window.AgentContact = App;
\ No newline at end of file
+// window.AgentContact = App;
```
Comment (PlasmaIntec) [inline client/src/index.jsx]: Is this comment still needed?

File: server/index.js [hunk 2]
```diff
@@ -106,26 +73,6 @@ app.delete('/houses/:houseId', (req, res) => {
...
-});
-
-// delete
-app.delete('/agents/:agentId', (req, res) => {
-  // pass in agent id
-});
-
 app.listen(PORT, () => {
   console.log(`Hidey-Ho Cap'n, we are now serving on port ${PORT}!`);
 });
```
Comment (PlasmaIntec) [inline server/index.js]: I would recommend moving the routes  into a controller.js file to adhere to the MVC Framework

Comment (elizabethcan) [reply server/index.js]: @PlasmaIntec I refactored if you'd like to take a look

Comment (PlasmaIntec) [reply server/index.js]: @elizabethcan changes look good- although is your database performing well even without pooling connections?

File: data/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+const { Client } = require('pg');
+
+const client = new Client({
+  // user: '',
+  host: 'localhost',
```
Comment (pon-y) [inline data/index.js:5]: Remove credentials to another file to not add credentials to git

**Review by pon-y — APPROVED** [review]

File: data/index.js (hunk 3 above)
Comment (elizabethcan) [reply data/index.js:5]: I have no credentials, but if I did, I would do that :)

**Pull request merged**